operations_on_datastructures/circular_queue_using_array.cpp

```diff
@@ -1,57 +1,97 @@
 #include <iostream>
 using namespace std;
```

Comment:
using namespace std is bad practice and hence should be removed from the repository